meson_options.txt

```diff
@@ -1 +1,2 @@
-option('gtkver', type: 'integer', value: 3, description: 'GTK+ version')
\ No newline at end of file
+option('gtkver', type: 'integer', value: 3, description: 'GTK+ version')
+option('docs', type: 'boolean', value: 'false', description: 'add html documentation and examples')
```

Comment:
`'false'` or `false`?

Reply:
Ok, I'll try your suggestions. Still new at `meson` but I do think docs should be an option to install.

Reply:
What I mean is, `'false'` is a string and `false` is a boolean, it's weird if the former works

Reply:
Nothing to add to @dimkr's review.

Reply:
Yeah, the 'former' did work.. weird!

Reply:
Anyway force pushed the correct stuff.